tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -15,6 +16,12 @@
 
 def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:
     ws.catalogs.list.return_value = [CatalogInfo(name="catalog1")]
+
+    def raise_catalog_exists(catalog: str, *_, **__) -> None:
+        if catalog == "catalog1":
+            raise BadRequest("Catalog 'catalog1' already exists")
+
+    ws.catalogs.create.side_effect = raise_catalog_exists
```

Comment:
If we don't need to test the value of `catalog`, as a convenience we can just provide the `BadRequest` instance as the side-effect and pytest will raise it.

Reply:
it is more accurate to only raise the error for `catalog1`